### PR TITLE
Add taxonomies to high boost fields

### PR DESCRIPTION
**Pull request by murilovmachado**
## Description
This PR adds taxanomies to high boost fields.

### Related Issues
Fixes https://tech-matters.atlassian.net/browse/CHI-2212

### Verification steps
Check that Resource Search in `aselo development` behaves slightly differently than `KHP staging`

### CODE REVIEW

File: resources-domain/packages/resources-search-config/tests/convertDocumentResources.test.ts [hunk 1]
```diff
@@ -38,6 +38,7 @@ describe('convertIndexDocument', () => {
         { key: 'description', value: 'This is the description', language: '' },
         { key: 'keywords', value: 'keyword1', language: '' },
         { key: 'keywords', value: 'keyword2', language: '' },
+        { key: 'taxonomies/taxonomy1', value: 'taxonomy1', language: '' },
```
Comment (murilovmachado) [inline resources-domain/packages/resources-search-config/tests/convertDocumentResources.test.ts:41]: I think this test is not being run currently.

Comment (stephenhand) [reply resources-domain/packages/resources-search-config/tests/convertDocumentResources.test.ts:41]: If you move the file from `resources-domain/packages/resources-search-config/tests/convertDocumentResources.test.ts` to  `resources-domain/packages/resources-search-config/tests/unit/convertDocumentResources.test.ts` they should start running

**Comment by stephenhand** [comment]
Should we also increase the name match weighting?

**Review by stephenhand — APPROVED** [review]

**Comment by murilovmachado** [comment]
@stephenhand pinging you again to take a second look. I've followed your advice and also bumped name (and id) weighting. Other than that, the most important changes you can see in the test file.

**Review by stephenhand — APPROVED** [review]
Thx for the comments - did you delete the test file in the old location?

**Pull request merged**